python/cudf/cudf/_fuzz_testing/tests/fuzz_test_orc.py

```diff
@@ -74,7 +74,7 @@ def orc_reader_stripes_test(input_tuple, columns, stripes):
     data_handle=OrcWriter,
     params={
         "compression": [None, "snappy"],
-        "enable_statistics": [True, False],
+        "enable_statistics": ["NONE", "ROWGROUP"]
```

Comment:
Similar here, would be better to vary between all three values.

Reply:
Done. Added `STRIPE` to the mix.